src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -255,6 +251,16 @@ public String getHelpText() {
     return getInfoText();
   }
 
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof PropertiesDiceRoller && getDisplayName().equals(((IBean) other).getDisplayName());
```

Comment:
It looks like `PropertiesDiceRoller#getDisplayName()` may return `null` if the `name` property is missing from the properties file.  I don't see anything in the code that requires that property to be present when creating an instance of `PropertiesDiceRoller`.  Since you're already importing `Objects`, I'd recommend you just write that subexpression as

```java
Objects.equals(getDisplayName(), ((IBean) other).getDisplayName())
```